eva-accession-ws/src/main/java/uk/ac/ebi/eva/accession/ws/rest/ClusteredVariantsRestController.java

```diff
@@ -52,22 +55,37 @@ public ClusteredVariantsRestController(
         this.submittedVariantsService = submittedVariantsService;
     }
 
-    @ApiOperation(value = "Find clustered variants by identifier")
+    @ApiOperation(value = "Find clustered variants (RS) by identifier", notes = "This endpoint returns an accessioned" +
+            " clustered variant (RS). See " +
+            "https://github.com/EBIvariation/eva-accession/wiki/Import-accessions-from-dbSNP#clustered-variant-refsnp-or-rs " +
+            "for an explanation of each field.")
```

Comment:
Suggested rewording:

> This endpoint returns the clustered variants (RS) represented by the given identifiers. For a description of the response, see https://github.com/EBIvariation/eva-accession/wiki/Import-accessions-from-dbSNP#clustered-variant-refsnp-or-rs

I find easier to read sentences when the link is at the end.